django/cantusdb_project/main_app/models/source.py

```diff
@@ -23,7 +23,7 @@ class Source(BaseModel):
     # Here in the new Cantus, we only use one field, and there are two levels: published and unpublished.
     # Published sources are available to the public. 
     # Unpublished sources are hidden from the list and cannot be accessed by URL until the user logs in.
-    published = models.BooleanField(blank=False, null=False)
+    published = models.BooleanField(blank=False, null=False, default=False)
```

Comment:
without this, the source-create page is broken. Is there ever a situation where we want a user to be able to create a source that is published=True right off the bat? If so, we should add a boolean field to the source-create page.

Reply:
The sources should be unpublished by default. I don't think we need to allow users to create an immediately-published source.